UnitTests/BraintreeCoreTests/BTHTTP_Tests.swift

```diff
@@ -749,6 +749,36 @@ final class BTHTTP_Tests: XCTestCase {
             }
         }
     }
+    
+    func testURLSessionTaskDidFinishCollectingMetrics() {
+        let mockDelegate = MockNetworkTimingDelegate()
+        http?.networkTimingDelegate = mockDelegate
+        
+        var originalRequest = URLRequest(url: URL(string: "https://example.com/graphql")!)
+        originalRequest.httpBody = """
+            {
+                "operationName": "TestMutation"
+            }
+            """.data(using: .utf8)
+        let task = testURLSession.dataTask(with: originalRequest)
+
+        let transactionMetrics = MockURLSessionTaskTransactionMetrics()
+        transactionMetrics.mockConnectStartDate = Date()
+        transactionMetrics.mockFetchStartDate = Date()
```

Comment:
just curious, how are these two values different? 👀

Reply:
> FetchStartDate: The time when the task started fetching the resource, from the server or locally.
> ConnectStartDate: The time immediately before the task started establishing a TCP connection to the server.

Apple has an amazing graphic in their documentation that would explain it better. [Apple doc reference](https://developer.apple.com/documentation/foundation/urlsessiontasktransactionmetrics)